src/main/java/org/apache/maven/shared/utils/logging/MessageUtils.java

```diff
@@ -98,7 +98,14 @@ private static void doSystemUninstall()
     {
         if ( JANSI )
         {
-            AnsiConsole.systemUninstall();
+            try
```

Comment:
Why is there a try-catch block required here, while it is not there for `AnsiConsole.systemInstall()`.
If this is an issue, it should be fixed in JAnsi (fix it at the root, instead of handling the symptom)

Reply:
This was already proposed here https://issues.apache.org/jira/browse/MNG-7161 but it seemed that the consensus was to go with that approach, which makes `MessageUtils` less dependent on changes on Jansi which caused this error. Fixing the root cause will fix _this_ specific use case, but there may be others which will break too, and as the `systemUninstall` is called as a last step before Maven finishes the build we can assume that it's safe to swallow these errors. I created https://github.com/fusesource/jansi/issues/214 to fix the root cause.

IMO we should not add it to the `systemInstall` as there you'd like to be aware when we can't install the color support and get the exception during runtime, as if we would swallow that error too, it will be harder to debug the root cause without any exception thrown. If you prefer to have the same handling, then I can add this.